### ISSUE TITLE
First time startup of the game should enforce high graphical settings, not low.

### ISSUE THREAD

**Issue by mcNuggets1** [Facepunch/garrysmod-requests#1675]
As the game is from 2004, the graphical settings can all be set to very high / ultra with no impact at all.
The only differences in performance with high vs low graphic settings were with HDR and Bloom.

Otherwise everything should be upped when initialising the game the first time.
Alot of new players don't know how to configure graphics or suspect the game to automatically select the highest possible (like in other games) and don't change anything, having their game ultimately looking like shit.

I would propose to up the settings by default, so players stop complaining about gmod having too bad graphics or asking for a graphics update.

**Comment by robotboy655** [comment]
![image](https://user-images.githubusercontent.com/3299036/89322873-92cf1880-d68d-11ea-93c8-a03fab01b62d.png)

Doesn't it default to the settings with the * on them by default?

**Comment by mcNuggets1** [comment]
Yes, which is anistrophic on x0, antialias disabled, hdr on for some stupid reason, textures only on high, and fov like 80? Instead of 90.

**Comment by viral32111** [comment]
Is it possible that whatever detects the recommended settings is a bit out of date for more modern computer hardware?

**Comment by mcNuggets1** [comment]
I would appreciate if this was considered.
The game is getting older every day and the modern graphics are far above the garry's mod standards.
With high enforced, the game runs with the same framerat as on low, even for bad computers.
I couldn't notice any difference, no matter which graphics setting I adjusted.

I would highly recommend to bring the grahpics up, to atleast look 2010-ish.

**Comment by mcNuggets1** [comment]
This would still benefit the game in a great way.
The settings turned to low don't even affect performance on modern computers nowadays in a all too bad way.

**Comment by wrefgtzweve** [comment]
<img width="164" height="176" alt="Image" src="https://github.com/user-attachments/assets/cba4ea8d-7ecb-4343-a298-acb1e0df7205" />

<img width="155" height="124" alt="Image" src="https://github.com/user-attachments/assets/bd650fa9-8b9d-4150-8e8e-7bacabcaa371" />

<img width="156" height="102" alt="Image" src="https://github.com/user-attachments/assets/64acfc74-6865-4717-a65f-46b35df39f73" />

@robotboy655 would it be possible to increase these as defaults?
I see so many people that never touch this menu and end up with awful looking game settings (usually when they post screenshots)
Gmod should really force these values to max as even the lowest ends of gpus can handle these settings nowadays.

**Comment by robotboy655** [comment]
With the recent fixes to dxsupport stuff it should default to higher settings now.
